### ISSUE TITLE
[SEV2] Keyboard focus is not visible on Queries, Graph, and Table under the Outline section

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#116516]
**Steps to reproduce**:
1. Select New Dashboard.  
2. Navigate to "Add visualization" button and hit ENTER key. "Select data source" popup will appear. 
3. Navigate to the TestData list item select it. 
4. Now navigate to the tool bar and select back to dashboard control and select. 
5. Navigate to the add menu present in tool bar and select add row control. 
6. Navigate to the  Explore  list item present in the New panel more action menu under row title. 
7. Navigate all controls present in Explore Pane. 
8. Verify whether the keyboard focus on  Queries, Graph, and Table under the Outline section is visible or not. 

**Actual Result**:

Keyboard focus does not appear on Queries, Graph, and Table under the Outline section. Users navigating through the interface using the Tab key cannot visually identify the focused elements. 
Observation: 
The keyboard focus indicator for Queries, Graph, and Table under the Outline section is either missing or not visible. This prevents users from perceiving which element is currently focused, especially in cases where the focus indicator is obscured or lacks sufficient contrast. 

**Expected Result**

The expected result is that the keyboard focus indicator should be clearly visible on Queries, Graph, and Table under the Outline section. The focus indicator must provide a persistent and perceivable visual cue, such as an outline, background color change, or another glyph, ensuring users can identify the currently focused element. 

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
WCAG: [2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html)
Section 508: [E207.2 WCAG Conformance](https://www.access-board.gov/ict/#E207.2), [E205.4 Accessibility Standard](https://www.access-board.gov/ict/#E205.4) and [E205.4.1 Word Substitution when Applying WCAG to Non-Web Documents](https://www.access-board.gov/ict/#E205.4.1), and [E207.2.1 Word Substitution when Applying WCAG to Non-Web Software](https://www.access-board.gov/ict/#e207-2-1-word-substitution-when-applying-wcag-to-non-web-softwar).
EN 301 549: [PDF of EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), which applies WCAG principles for Focus Visible to 9.2.4.7 web, 10.2.4.7 non-web documents, and 11.2.4.7 software.

**Environment**:
- Grafana version: 12.2.0
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by JonCole** [comment]
We consider this a Sev 2 and ask that you fix within 60 days

**Comment by v-kavyagupta** [comment]
We verified this issue on URL:http://localhost:3000/explore?schemaVersion=1&panes=%7B%2244t%22%3A%7B%22datasource%22%3A%22ffil5e303c3k0f%22%2C%22queries%22%3A%5B%7B%22scenarioId%22%3A%22random_walk%22%2C%22seriesCount%22%3A1%2C%22refId%22%3A%22A%22%2C%22datasource%22%3A%7B%22type%22%3A%22grafana-testdata-datasource%22%2C%22uid%22%3A%22ffil5e303c3k0f%22%7D%7D%5D%2C%22range%22%3A%7B%22from%22%3A%22now-1h%22%2C%22to%22%3A%22now%22%7D%2C%22compact%22%3Afalse%7D%7D&orgId=1
Issue is still repro, Keyboard focus is not visible on Queries, Graph, and Table under the Outline section.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/8d57b925-06d6-4820-b423-b10604185bcc" />
